src/open_inwoner/js/components/search/index.js

```diff
@@ -11,15 +11,17 @@ const radioButtons = document.querySelectorAll(
   })
 })
 
-var timerId = 0
+let timerId = null
 
 const searchForm = document.getElementById('search-form')
 
 const filterButtons = document.querySelectorAll('.filter .checkbox__input')
 ;[...filterButtons].forEach((checkbox) => {
   checkbox.addEventListener('change', (event) => {
-    clearInterval(timerId)
-    timerId = setInterval(() => {
+    clearTimeout(timerId)
+
+    // Set a new interval
```

Comment:
Should be "Set a new timeout", since you've replaced `clearInterval` with `clearTimeout` following Bart's suggestion.